### DISCUSSION TITLE
[PSD6] Example 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#146] [Esercizi (A-L)]
$Y = (A + BC)(A + DE)$

**Apply T8’ first when possible:** $W+XZ = (W+X)(W+Z)$

**Comment by Elia-Belli** [comment]
Ritengo più veloce questo prodecimento:
$(A+BC)(A+DE)$
$A + ADE + ABC + BCDE$
$A + BCDE$

**Comment by FeddyLix17** [comment]
@Elia-Belli è giusto pure come hai fatto te, sono efficaci entrambi i procedimenti
